mlir/lib/Conversion/GPUCommon/GPUOpsLowering.cpp

```diff
@@ -194,18 +194,22 @@ GPUFuncOpLowering::matchAndRewrite(gpu::GPUFuncOp gpuFuncOp, OpAdaptor adaptor,
   // Get memref type from function arguments and set the noalias to
   // pointer arguments.
   for (const auto &en : llvm::enumerate(gpuFuncOp.getArgumentTypes())) {
```

Comment:
nit: while we are at this, we could make this a bit more readable 
```suggestion
  for (auto [idx, argType] : llvm::enumerate(gpuFuncOp.getArgumentTypes())) {
```
(this is existing code, so feel free to leave as-is)

Reply:
Done, thanks!